hasjob/models/jobpost.py

```diff
@@ -587,6 +588,17 @@ def viewstats_helper(jobpost_id, interval, limit, daybatch=False):
     cviewed = batches * [0]
     copened = batches * [0]
     capplied = batches * [0]
+    cbuckets = batches * ['']
+
+    now_local = datetime.now()
```

Comment:
Always use `utcnow()`. You can't figure out the client's local time on the server. You'll have to cast times in the labels back into the user's timezone using `g.user.tz`, defaulting back to `app.config['tz']`.